### PR TITLE
check if pipe is served by administrators alias

### PR DESCRIPTION
**Pull request by zanderz**
Check it out, @maxtaco , I think this might fix it. Will have some users test a build with this first though.

Here's the relevant debug from a session which fails without this change:
```
2018-06-05T08:39:53.256982-07:00 - [DEBU keybase pipeowner_windows.go:98] 010 Pipe account: Administrators, BUILTIN, 4, <nil>
2018-06-05T08:39:53.259974-07:00 - [DEBU keybase pipeowner_windows.go:99] 011 User account: Steve, SteveSamsung9, 1, <nil>
```

### CODE REVIEW

File: go/libkb/pipeowner_windows.go [hunk 1]
```diff
@@ -85,10 +86,21 @@ func IsPipeowner(name string) (bool, error) {
...
 			return false, err // return original busy error
 		}
-		fileSid, err = GetFileUserSid(name)
+		pipeSid, err = GetFileUserSid(name)
 	}
 	if err != nil {
 		return false, err
 	}
-	return windows.EqualSid(fileSid, userSid), nil
+	isEqual := windows.EqualSid(pipeSid, userSid)
```
Comment (maxtaco) [inline go/libkb/pipeowner_windows.go]: Can you throw some more debugging in right here too?

File: go/libkb/pipeowner_windows.go [hunk 2]
```diff
@@ -68,13 +69,13 @@ func GetFileUserSid(name string) (*windows.SID, error) {
 	return userSID, nil
 }
 
-func IsPipeowner(name string) (bool, error) {
+func IsPipeowner(log logger.Logger, name string) (bool, error) {
```
Comment (maxtaco) [inline go/libkb/pipeowner_windows.go]: it would be nice to put debugs on the entry and exit of this function.

Comment (zanderz) [reply go/libkb/pipeowner_windows.go]: I don't think that helps us but will do

File: go/libkb/pipeowner_windows.go (hunk 1 above)
Comment (zanderz) [reply go/libkb/pipeowner_windows.go]: The caller always prints out that error someplace, do you want it debug logged here too, or...?

**Comment by maxtaco** [comment]
ok, whatever logging you can, to hunt down further problems here

also, failed CI due to `go fmt` on `socket_windows`

**Pull request merged**